TweetMate/src/main/java/com/ybsystem/tweetmate/utils/StorageUtils.java

```diff
@@ -106,7 +108,7 @@ public static boolean isPermitted(Context context) {
      * Request storage permission
      */
     public static void requestPermission(Activity activity) {
-        ToastUtils.showShortToast("アクセス許可が必要です。");
+        ToastUtils.showShortToast(STR_FAIL_NEED_PERMISSION);
         new Handler().postDelayed(() -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1602)

Reply:
Fixed: https://github.com/ybs32/TweetMate-for-Android/pull/85/commits/0a5013e538213bacb2774f24bfa4ed6959fa720c